src/client/src/consumer.rs

```diff
@@ -411,7 +413,19 @@ impl Record {
     }
 
     /// The actual bytes for this record.
-    pub fn try_into_bytes(self) -> Option<Vec<u8>> {
+    pub fn into_bytes(self) -> Vec<u8> {
         self.record.value.inner_value()
     }
+
+    /// The actual bytes for this record.
+    pub fn as_bytes(&self) -> &Vec<u8> {
+        self.record.value.inner_value_ref()
+    }
+}
+
+impl TryFrom<Record> for String {
```

Comment:
We may as well also have `impl From<Record> for Vec<u8>` which is the same as `into_bytes`